microsoftgraph/microsoft-graph-explorer-v2
# Fix: add version select

## Overview

Adds a version selector option
The version selection changes automatically when you select queries from the sidebar.

Improves #205 
Fixes #200 

### Demo

![image](https://user-images.githubusercontent.com/53429535/66748515-06e6c880-ee90-11e9-8636-0b8e03428f13.png)


## Testing Instructions

* Change the version from v1.0 to beta and notice how the query url changes
* Click on a url in the queries and notice the change in the version selector
